pennylane/transforms/commutation_dag.py

```diff
@@ -1043,6 +896,7 @@ def size(self):
         """
         return len(self._multi_graph)
 
+    # pylint: disable=no-member
```

Comment:
Why is this needed?

Reply:
I was getting a pylint warning about `dot.write_png`.

Reply:
Could you maybe move it to the line where `dot.write_png` is? Doing this we avoid disabling the pylint check for the whole function.